pkg/controller/pkg/manager/reconciler.go

```diff
@@ -47,6 +47,10 @@ const (
 	pullWait      = 1 * time.Minute
 )
 
+var (
+	blockOwnerDeletion = true
+)
```

Comment:
Nit: Could this be a function local var? https://godoc.org/k8s.io/utils/pointer#BoolPtr could work too.

Reply:
Good call, done :+1: Was actually thinking about adding this to the helpers for owner refs we have in crossplane-runtime, but introducing a function that only sets a single field seems like over-optimizing